portal-knights.kvp

```diff
@@ -0,0 +1,83 @@
+Meta.Arch=x86_64
+Meta.Author=IceOfWraith
+Meta.URL=https://store.steampowered.com/app/374040/Portal_Knights/
+Meta.DisplayImageSource=steam:374040
+Meta.EndpointURIFormat=steam://connect/{0}
+Meta.ConfigManifest=portal-knightsconfig.json
+Meta.MetaConfigManifest=portal-knightsmetaconfig.json
+Meta.ConfigRoot=portal-knights.kvp
+Meta.MinAMPVersion=
+Meta.SpecificDockerImage=cubecoders/ampbase:wine
+Meta.DockerRequired=False
+Meta.ContainerPolicy=RecommendedOnLinux
+Meta.Prerequsites=[]
+Meta.ConfigReleaseState=NotSpecified
+App.DisplayName=Portal Knights
+App.RootDir=./portal-knights/
+App.BaseDirectory=./portal-knights/374040/
+App.ExecutableWin=374040\pk_dedicated_server.exe
+App.ExecutableLinux=/usr/bin/wine
+App.WorkingDir=374040
+App.LinuxCommandLineArgs="./pk_dedicated_server.exe"
+App.WindowsCommandLineArgs=
+App.CommandLineArgs={{$PlatformArgs}} -config "{{$FullBaseDir}}server_config.json" -log Server.log
+App.UseLinuxIOREDIR=False
+App.AppSettings={}
+App.EnvironmentVariables={"LD_LIBRARY_PATH":"./linux64:%LD_LIBRARY_PATH%","SteamAppId":"374040","STEAM_COMPAT_DATA_PATH":"{{$FullRootDir}}1580130","STEAM_COMPAT_CLIENT_INSTALL_PATH":"{{$FullRootDir}}1580130"}
+App.CommandLineParameterFormat=-{0} {1}
+App.CommandLineParameterDelimiter= 
+App.ExitMethod=OS_CLOSE
+App.ExitTimeout=30
+App.ExitString=stop
+App.ExitFile=app_exit.lck
+App.HasWriteableConsole=True
+App.HasReadableConsole=True
+App.SupportsLiveSettingsChanges=False
+App.LiveSettingChangeCommandFormat=set {0} "{1}"
+App.ApplicationIPBinding=0.0.0.0
+App.Ports=[{"Protocol":2,"Port":28001,"Offset":0,"Range":1,"Ref":"GamePort","Name":"Game Port","Description":"Main game port","ChildPorts":null}]
+App.AdminPortRef=
+App.PrimaryApplicationPortRef=GamePort
+App.UniversalSleepApplicationUDPPortRef=
+App.UniversalSleepSteamQueryPortRef=
+App.MaxUsers=4
+App.UseRandomAdminPassword=False
+App.RemoteAdminPassword=Password123
+App.AdminMethod=STDIO
+App.AdminLoginTransform=None
+App.RCONConnectDelaySeconds=5
+App.RCONConnectRetrySeconds=5
+App.TelnetLoginFormat={0}
+App.UpdateSources=[{"UpdateStageName": "SteamCMD Download","UpdateSourcePlatform": "All", "UpdateSource": "SteamCMD", "UpdateSourceData": "374040", "ForceDownloadPlatform": "Windows"},{"UpdateStageName": "Server Extract","UpdateSourcePlatform": "Windows", "UpdateSource": "Executable", "UpdateSourceData": "cmd.exe", "UpdateSourceArgs": "/C cd portal-knights\\374040 && tar -xf dedicated_server.zip"},{"UpdateStageName": "Server Extract","UpdateSourcePlatform": "Linux", "UpdateSource": "Executable", "UpdateSourceData": "/bin/bash", "UpdateSourceArgs": "-c 'cd portal-knights/374040 && unzip -u dedicated_server.zip'"},{"UpdateStageName":"Config File Download","UpdateSourcePlatform":"All","UpdateSource":"FetchURL","UpdateSourceData":"https://github.com/CubeCoders/AMPTemplates/main/portal-knightsserver_config.json","UpdateSourceTarget":"{{$FullBaseDir}}","UpdateSourceArgs":"server_config.json","OverwriteExistingFiles":false}]
```

Comment:
Please add DeleteAfterExtract

Reply:
I planned to leave it since it's a file included in the Steam download. If it's deleted then Steam will redownload it every time an update is performed even if there's no new version.

Reply:
Got it, that makes sense. Dumb server design